notebooks/MAST/HSC/CASJOBS/HCV_API/HCV_API_demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+    "nova_606 = ascii.read(hcvsearch(table='hcv', MatchID=matchid, Filter='ACS_F606W'))\n",
+    "print(f\"{(time.time()-t0):.1f} sec: retrieved {len(nova_606)} F606W measurements\")\n",
+    "\n",
+    "nova_814 = ascii.read(hcvsearch(table='hcv', MatchID=matchid, Filter='ACS_F814W'))\n",
+    "print(f\"{(time.time()-t0):.1f} sec: retrieved {len(nova_814)} F814W measurements\")\n",
+    "\n",
+    "# get the object RA and Dec as well\n",
+    "nova_tab = ascii.read(hcvsearch(table='hcvsummary', MatchID=matchid, Filter='ACS_F814W'))\n",
+    "print(f\"{(time.time()-t0):.1f} sec: retrieved object info\")\n",
+    "\n",
+    "nova_606"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.rcParams.update({'font.size': 16})\n",
+    "plt.figure(1, (15, 10))\n",
+    "\n",
+    "x = nova_606['MJD']\n",
+    "y = nova_606['CorrMag']\n",
+    "e = nova_606['MagErr']\n",
+    "plt.errorbar(x, y, yerr=e, fmt='ob', ecolor='k', elinewidth=1, markersize=8, label='ACS F606W')\n",
+    "\n",
+    "x = nova_814['MJD']\n",
+    "y = nova_814['CorrMag']\n",
+    "e = nova_814['MagErr']\n",
+    "plt.errorbar(x, y, yerr=e, fmt='or', ecolor='k', elinewidth=1, markersize=8, label='ACS F814W')\n",
+    "\n",
+    "plt.gca().invert_yaxis()\n",
+    "plt.xlabel('MJD [days]')\n",
+    "plt.ylabel('magnitude')\n",
+    "plt.legend(loc='best', title=f'Nova in M87 (MatchID: {matchid})')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get HLA image cutouts for the nova <a name=\"cutouts\"></a>\n",
+    "\n",
+    "The [Hubble Legacy Archive (HLA)](https://hla.stsci.edu) images were the source of the measurements in the HSC and HCV, and it can be useful to look at the images.  Examination of the images can be useful to identified cosmic-ray contamination and other possible image artifacts.  In this case, no issues are seen, so the light curve is reliable.\n",
+    "\n",
+    "Note that the ACS F606W images of M87 have only a single exposure, so they do have cosmic ray contamination.  The accompanying F814W images have multiple exposures, allowing CRs to be removed.  In this case the F814W combined image is used to find objects, while the F606W exposure is used only for photometry.  That reduces the effects of F606W CRs on the catalog but it is still a good idea to confirm the quality of the images.\n",
+    "\n",
+    "The `get_hla_cutout` function reads a single cutout image (as a JPEG grayscale image) and returns a PIL image object.  See the documentation on the [fitscut image cutout service](http://hla.stsci.edu/fitscutcgi_interface.html) for more information on the web service being used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_hla_cutout(imagename, ra, dec, size=33, autoscale=99.5, asinh=True, zoom=1):\n",
```

Comment:
It seems odd that this function is here while all the ones above are in a single block. I think it *does* make sense to do that here since `get_hla_cutout` is only used in this one place, but that should be explained, rather than just left without explanation to the uninformed reader.

Reply:
I presume the notebook was written this way because the `Useful functions` are interrelated. `def get_hla_cutout` is a separate unrelated function.